workflows/rnaseq/main.nf

```diff
@@ -782,6 +782,127 @@ workflow RNASEQ {
         ch_multiqc_report = MULTIQC.out.report
     }
 
+    publish:
+    QUANTIFY_STAR_SALMON.out.results                        >> 'star_salmon/'
```

Comment:
I think this is going to be annoying to users/developers not knowing where a publish path is defined when trying to debug why their file is being written to another location.

Reply:
The main principle is that callers overwrite callees. But also I think we will extend the `inspect` command to also show the resolved publish targets for the entire pipeline, so that it is more clear.